core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-traffic/dot-experiments-configuration-traffic.component.spec.ts

```diff
@@ -76,4 +76,26 @@ describe('DotExperimentsConfigurationTrafficComponent', () => {
 
         expect(store.openSidebar).toHaveBeenCalledOnceWith(ExperimentSteps.TRAFFIC);
     });
+
+    it('should disable button and show tooltip when experiment is nos on draft', () => {
```

Comment:
Do we have any test to check if the tooltip in draft is showing?

Reply:
done